.github/ISSUE_TEMPLATE/roadmap-item.md

```diff
@@ -0,0 +1,28 @@
+---
+name: 'Roadmap Item'
+about: Create roadmap item for this project
+title: Roadmap Item =description=
+labels: kind/roadmap
+assignees: ''
+
+---
+
+# O3DE Roadmap Item Template
+
+### When using this template, you do not have to fill out every question below. The questions are there for guidance.
+
+This roadmap item template should be used for any feature that shows in the O3DE public roadmap. The issue communicates the initiative and vision of the SIG.
+
+# ----- DELETE EVERYTHING FROM THE TOP TO THE SUMMARY LINE BELOW WHEN USING TEMPLATE ----- #
+
+### Summary:
+Single paragraph explanation of the roadmap item
+
+### What is the relevance of this feature?
+What problems does it solves? Why is this important? What are the use cases? What will it do once completed?
```

Comment:
Do we want to owners to discuss impact of feature here? If feature is expected to change multiple system do we want to capture that early.

Reply:
I don't have any concerns to add that prompt to this question. Do you have any concerns? @tonybalandiuk

Reply:
I'll add the question in the section.

Reply:
Done. Please review @lmbr-pip